### ISSUE TITLE
[UI] Use consistent casing for app path in IDC_LISTVIEW

### ISSUE THREAD

**Issue by xEBFE** [henrypp/simplewall#90]
Version: 2.0.20, 64-bit (Unicode) on Windows 7

If I add a file via "Add file..." (e.g., C:\Users\Me\AppData\Local\FluxSoftware\Flux\flux.exe) or add a process via "Add process" (e.g., flux.exe), the app path's casing in the IDC_LISTVIEW is consistent with how Windows would display the full file path.

When simplewall detects attempted network activity and automatically adds the app path of the process which attempted to communicate to the IDC_LISTVIEW, I've noticed that it always lower-cases the entire app path (including file name) with the exception of the drive letter. While the app path, regardless of case, points to correct location of the file and/or process could you correct the app's path casing to be consistent with how Windows would display the full file path.

Basically, instead of automatically adding upon network detection:

C:\users\me\appdata\local\fluxsoftware\flux\flux.exe

Could you have it be automatically added as:

C:\Users\Me\AppData\Local\FluxSoftware\Flux\flux.exe

As this would mirror the behavior of manually adding a file or process from the right-click pop-up menu and would look more aesthetically pleasing by being consistent.

**Comment by henrypp** [comment]
No. Case correction for auto-added applications will not be implemented.
